integration-tests/src/test/resources/apps/buildDeployCoherenceAppInPod.sh

```diff
@@ -75,11 +74,11 @@ cres=$?
 
 echo "Deploy ${APP_NAME} ear using curl:"
 
-echo -e "curl --noproxy '*' --silent  --user ${USER}:${PASSWORD} -H X-Requested-By:MyClient -H Accept:application/json -H Content-Type:multipart/form-data -F \"model={ name: '${APP_NAME}', targets: [ { identity: [ clusters, '${EAR_DEPLOY_TARGET}' ] } ] }\" -F \"sourcePath=@${ARCHIVE_FILE_EAR}\" -H \"Prefer:respond-async\" -X POST http://${HOST}:${PORT}/management/weblogic/latest/edit/appDeployments -o ${APP_DIR_INPOD}/deployear.out\n"
+echo -e "curl --noproxy '*' -v --user ${USER}:${PASSWORD} -H X-Requested-By:MyClient -H Accept:application/json -H Content-Type:application/json -d "{name: '${APP_NAME}', sourcePath='${ARCHIVE_FILE_EAR}', targets: [ { identity: [ 'clusters', '${EAR_DEPLOY_TARGET}' ] } ] }" -X POST http://${HOST}:${PORT}/management/weblogic/latest/edit/appDeployments -o ${APP_DIR_INPOD}/deployear.out\n"
```

Comment:
it is my strong preference that you do not duplicate this string (the command to be run) -- it will just lead to future mistakes or difficulty understanding the intent

Reply:
you would like me to remove the echo statements?

Reply:
no, those are useful, i just want you to do something like this pseudocode:
```
COMMAND="whatever the command is with all its arguments and stuff"
echo "Executing $COMMAND"
$COMMAND
```

Reply:
got it. Will fix it.